webpack/assets/javascripts/react_app/components/Layout/LayoutActions.js

```diff
@@ -57,3 +63,26 @@ export const changeLocation = location => dispatch => {
     },
   });
 };
+
+export const onLogout = (
```

Comment:
this is quite a lot of code just to change the click action from get to post. Why do we need to do this whole async/await dance just for a simple post action that returns very quickly (as logout doesn't do any heavy computation)?

Reply:
?

Reply:
current behaviour (after clicking yes on the logout confirmation) is a button which is inside a form

<img width="594" alt="Screenshot 2019-04-28 19 46 01" src="https://user-images.githubusercontent.com/24938324/56867480-54f53600-69ee-11e9-8030-b7749af456a2.png">

I dislike writing code like this in 2019, I agree redux means more code but I do see use cases for listening to a logout action in the future.
@sharvit your thoughts on this?

Reply:
The original behavior in the menu before refactoring for react was performing a POST action to log out the user which would reply with a redirect back to the login page.
Seems to me like writing over a hundred lines to replace 4 lines just in case we might want to do something about it in the future is quite a lot of overhead and complexity for something that should be quite simple?